syntax = "proto3";
import "action.proto";

package whiteboard;

message whiteboardPacket {
    uint32 version = 1;
    uint32 packet_type = 2;
    uint32 session_id = 3;   // Session ID
    uint32 user_id = 4;      // User ID
    whiteboard.PacketAction action = 5; // Payload
}